Marvelys-Torres/tarea-5/index.js

```diff
@@ -0,0 +1,201 @@
+class Caja {
```

Comment:
@MarvelysTorres each CLass should be on its own file, like State.js also should be named in english